cluster-autoscaler/cloudprovider/kubemark/kubemark_linux.go

```diff
@@ -13,6 +13,10 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
+// +build linux
```

Comment:
I think you need an empty line before this, otherwise the verify-boilerplate script gets unhappy.